component/src/main/java/io/siddhi/extension/store/elasticsearch/ElasticsearchEventTable.java

```diff
@@ -846,7 +846,6 @@ private void createIndex() {
             XContentBuilder builder = XContentFactory.jsonBuilder();
             builder.startObject();
             {
```

Comment:
We can remove the additional enclosing brackets too. Ref: line 848 and 886

Reply:
Thanks for pointing that out. Fixed in 8683f9d